.github/workflows/pull-request.yml

```diff
@@ -109,6 +109,10 @@ jobs:
       - name: apt get
         run: sudo apt-get update &&
              sudo apt-get -y install valgrind alsa-utils libasound2-dev
+             octave octave-io octave-signal
+
+      - name: docker
+        run: docker pull thesofproject/sof && docker tag thesofproject/sof sof
```

Comment:
I just removed docker here, please don't re-add it unless it's required again for some specific reason.

Reply:
Ah, I think I messed up when I resolve merge conflict... I will fix it